resolversrv/app.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from werkzeug.serving import run_simple
+
+from flask_restful import Api
+from flask_discoverer import Discoverer
+
+from adsmutils import ADSFlask
+
+from resolversrv.views import bp
+
+def create_app(**config):
+    """
+    Create the application and return it to the user
+    :return: flask.Flask application
+    """
+
+    # todo: figure out why getting this warning 'Sorry, cant find the proj home; returning the location of the caller'
+    if config:
+        app = ADSFlask(__name__, static_folder=None, local_config=config)
```

Comment:
if the service grows (which is very likely), then app.py should have its own version of the ADSFlask with its own methods, such as app.get_record(bibcode)

Reply:
you want to drive it from Flask or ADSFlask?